crates/storage/src/state.rs

```diff
@@ -76,11 +76,11 @@ impl L1StateTable {
         Ok(())
     }
 
-    /// Returns the [root](GlobalRoot) of the given block.
+    /// Returns the [root](StateCommitment) of the given block.
     pub fn get_root(
```

Comment:
If this now returns a `StateCommitment` maybe we should also rename the function to `get_state_commitment`?

Reply:
Cheers, I somehow missed this one :+1: